containers/flatpak/cockpit-client.yml.in

```diff
@@ -36,3 +36,33 @@ modules:
         sha256: @ARCHIVE_SHA256@
       - type: file
         path: @FLATPAK_ID@.releases.xml
+
+  - name: cockpit-podman
```

Comment:
This is fine (and welcome) for our demo of course, but note that I don't want to land this in production. See discussion in https://issues.redhat.com/browse/COCKPIT-943